solr/solrj/src/java/org/apache/solr/common/util/CommandOperation.java

```diff
@@ -223,7 +223,7 @@ public static List<CommandOperation> parse(InputStream in, Set<String> singleton
     List<CommandOperation> operations = new ArrayList<>();
 
     final HashMap<Object, Object> map =
-        new HashMap<>(0) {
+        new HashMap<>() {
```

Comment:
If we want to enforce the size 0 - this needs to move to an real class that we can add suppressforbidden to. We can't add suppressforbidden to an anonymous class.

Reply:
+1 to not keep 0. Anyway it is replaced internally by a power of 2.